ui/app/AppLayouts/Chat/controls/community/ExtendedDropdownContent.qml

```diff
@@ -30,38 +31,51 @@ Item {
     }
 
     function goBack() {
-        root.state = d.listView_depth1_State
+        d.reset()
     }
 
     function goForward(key, itemName, itemSource, subItems) {
         d.currentSubitems = subItems
         d.currentItemKey = key
         d.currentItemName = itemName
         d.currentItemSource = itemSource
-        root.state = d.listView_depth2_State
+        root.state = d.useThumbnailsOnDepth2
```

Comment:
Thanks for fixing that!! Nice to keep the thumbnails option / view if it has been previously selected! :)